Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartProblemsView.java

```diff
@@ -203,15 +207,18 @@ public void clearNotifications() {
   private void showNotification(@NotNull NotificationType notificationType,
                                 @NotNull String title,
                                 @Nullable String content,
-                                @Nullable Icon icon) {
+                                @Nullable Icon icon,
+                                boolean terse) {
     clearNotifications();
 
     if (myDisabledForSession) return;
 
     content = StringUtil.notNullize(content);
-    if (!content.endsWith("<br>")) content += "<br>";
-    content += "<br><a href='disable.for.session'>Don't show for this session</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;" +
-               "<a href='never.show.again'>Never show again</a>";
+    if (!terse) {
+      if (!content.endsWith("<br>")) content += "<br>";
+      content += "<a href='disable.for.session'>Don't show for this session</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;" +
```

Comment:
An empty line is added here on purpose, so I recovered it.

Reply:
I removed for a slightly more compact display, but your call :)